### ISSUE TITLE
Any way to avoid PSR-12's declare(strict_types=1) rules??

### ISSUE THREAD

**Issue by hopeseekr** [squizlabs/PHP_CodeSniffer#2932]
Hi,

I currently run `php vendor/bin/phpcs --standard=PSR12 src` in several of my projects' CI.

They have been failing for 6+ months, because my code is organized like:

```php
<?php declare(strict_types=1);

/**
 * This file is part of SimpleDTO, a PHP Experts, Inc., Project.
 *
 * Copyright © 2019 PHP Experts, Inc.
 * Author: Theodore R. Smith <theodore@phpexperts.pro>
 *  GPG Fingerprint: 4BF8 2613 1C34 87AC D28F  2AD8 EB24 A91D D612 5690
 *  https://www.phpexperts.pro/
 *  https://github.com/phpexpertsinc/SimpleDTO
 *
 * This file is licensed under the MIT License.
 */

namespace PHPExperts\SimpleDTO;
```

It currently generates several PHPCS warnings:

```
--------------------------------------------------------------------------------------------
FOUND 3 ERRORS AFFECTING 2 LINES
--------------------------------------------------------------------------------------------
 1 | ERROR | [x] Header blocks must be separated by a single blank line
 1 | ERROR | [x] Opening PHP tag must be on a line by itself
 3 | ERROR | [ ] The file-level docblock must follow the opening PHP tag in the file header
--------------------------------------------------------------------------------------------
```

Is there any way to keep the rest of the PSR-12 checks but not those?

I've added this as a [**StackOverflow question**](https://stackoverflow.com/questions/61105282/how-can-i-partially-disable-phpcs-psr-12-declarestrict-types-1-checks), if you'd prefer to answer it there.

**Comment by gsherwood** [comment]
Closing as it has been answered on SO.
